fpdf/table.py

```diff
@@ -223,25 +237,23 @@ def _render_table_cell(
         self,
         i,
         j,
-        row_height,
+        row_height,  # height of a row of text
         fill=False,
+        cell_height=None,  # height of cell for rendering borders and layout and images
```

Comment:
```suggestion
        cell_height=None,  # full height of a cell, including padding, used to render borders and images
```
Maybe the naming of those parameters is not the clearest...

What would you think of this alternate naming @RubendeBruin:
* row_height,  # full height of a cell, including padding, used to render borders and images
* text_height,  # height of a row of text

Reply:
You did not like `text_height` ?

Reply:
outdated